README.rst

```diff
@@ -71,7 +74,26 @@ sub-directory called ``baseline`` relative to the test files (this name
 is configurable, see below). You can also generate the baseline images
 directly in the right directory.
 
-You can then run the tests simply with::
+With a Hash Library
+^^^^^^^^^^^^^^^^^^^
+
+Instead of comparing to baseline images, you can instead compare against a json
+library of sha256 hashes. This has the advantage of not having to check baseline
+images into the repository with the the tests, or download them from a remote
```

Comment:
Double the

Reply:
Oh interesting I made the suggested edit in a pending review on my computer but the switched to phone and it looks like it remembered